packages/mui/src/date-time-pickers/wrapper.tsx

```diff
@@ -29,19 +34,38 @@ type PickerProps<
 type Props<P extends PickerTypes, In, Out> = SubsetPartial<
   PickerProps<P, In, Out>,
   'renderInput'
->;
+> &
+  (P extends typeof TimePicker ? unknown : { multiLocale?: boolean });
+
 export { Props as WrappedPickerProps };
 
 export function WrapPicker<P extends PickerTypes>(Picker: P) {
```

Comment:
add second param to enable or disable multiCalendar

Reply:
Ok!